docs/components/typography/index.html

```diff
@@ -199,14 +207,19 @@ <h2 id="keyboard">Keyboard</h2>
     {% endcode %}
   </section>
 
-  <section><!-- Code Block -->
+  <section data-visreg="typography-code-block"><!-- Code Block -->
     <h2 id="code-block">Code Block</h2>
+    <p>Example (python):</p>
 
     <div class="demo">
-      <pre><code>[Insert Code Here]</code></pre>
+      <pre><code>>>> print round(4 * sum(-1.0 ** (n%2) / (2*n + 1) for n in range(9999999)), 6))
```

Comment:
Please revert these changes and create an appropriate `<figure>` below alongside the other examples.